Bio/Wise/psw.py

```diff
@@ -85,10 +86,9 @@ def __init__(self, unit, column, kind):
         self.column = column
         self.kind = kind
 
-    def __str__(self):
-        return "ColumnUnit(unit=%s, column=%s, %s)" % (self.unit, self.column, self.kind)
-
-    __repr__ = __str__
+    def __repr__(self):
+        """Represent the ColumnUnit object as a string for debugging."""
+        return "ColumnUnit(unit=%r, column=%r, kind=%r)" % (self.unit, self.column, self.kind)
```

Comment:
As noted by Markus, using ``%r`` here to get quotes round the values, and make the output a valid fragment of Python. Also had to add the missing ``kind=`` bit too.

Reply:
Shouldn't have touched this without the command line tool dependencies installed locally - updating the tests via TravisCI failures is tedious.